modules/main-rules/pages/assault-moves-and-charges.adoc

```diff
@@ -4,26 +4,45 @@ During the Assault phase you may only move and attack with detachments that you
 * You must move these units towards the enemy -- but not necessarily towards the _nearest_ enemy.
-* You can move these units less than 5cm *only* where they charge into close combat (base contact) with enemy units that are within 5cm.
+* You can move these units less than 5 cm *only* where they charge into close combat (base contact) with enemy units that are within 5 cm.
 
 Otherwise, moves in the Assault phase are subject to all the rules we described for the Movement phase -- except that the Snap-fire rule does not apply in the Assault phase.
 
 == Charge
 
 As you move detachments in this phase, you may declare that any or all of them will *charge* so that they become engaged in close combat with the enemy.
+
 This is the only time that you may move your units into base contact with enemy units.
+And where you do move any individual unit into base contact with an enemy unit, we call this a *charge move*.
+
+* You don't have to declare and make a charge into close combat unless you want to, with one exception: If the 'must move towards the enemy' and 'must move at least 5 cm, or into base contact' requirements force you to place units in base contact, then you have made charge moves by definition and so you have a charge whether you like it or not!
+Of course you may move more units into base contact, where they can reach.
+* If you don't get at least one unit into base contact with an enemy then there are no charge moves by definition, which means that the detachment does not charge and there will be no close combat.
+(Though there will be a firefight if any units are within 15cm of the enemy.)
 
-* You don't have to declare and make a charge into close combat unless you want to, with one exception: If you can only meet the 'must move towards the enemy' requirement by moving toward an enemy that has units that are within 5cm of your units, then you must charge and place units in base contact.
-* For a charge, you may double the Speed value of *infantry* units *if* this will bring them into base contact with enemy units.
-* If you don't get at least one unit into base contact with an enemy then there is no charge and there will be no close combat.
-(Though there will be a firefight if any units are within 15cm of the enemy.)
\ No newline at end of file
+You can benefit from a special increase to the Speed value of some particular units when you make a charge move with them.
+(Remember, it is only a charge move if the units will get into base contact with the enemy -- you don't get the increase if they won't get into base contact.)
+
+* You may double the Speed value of any *Infantry* unit.
+* Where a unit has a split Speed value such as `20/30 cm`, you may *not* double the first value -- use the second value instead.
+* For any other unit, must use their Speed value in the usual way.
```

Comment:
This is ambiguous, and can be read as doubling the second value instead of doubling the first value.

I've also removed an errant 'must' from the next bullet.

```suggestion
* Where a unit has a split Speed value such as `20/30 cm`, use the second value *instead* of doubling the first value.
* For any other unit, use their Speed value in the usual way.
```

Reply:
Well spotted, I agree that it could be read that way, and indeed there is an errant 'must'.
Thinking about it, I don't think we really need to mention "don't double" because that only applies to the previous bullet anyway — strictly, we don't use the second value instead of double-Speed (which we wouldn't do anyway for Cavalry because they count as Vehicle not Infantry). Rather, we use the second value instead of the normal Speed (first value), as one of a couple of special cases for a charge move .
```suggestion
* Where a unit has a split Speed value such as `20/30 cm`, use the second value.
* For any other unit, use their Speed value in the usual way.
```

Reply:
Don't forget that this covers Jump Packs units as well, which as Infantry **do** default to getting double Speed. 

Maybe:

```suggestion
* Where a unit has a split Speed value such as `20/30 cm`, use the second value but you cannot double it.
* For any other unit, use their Speed value in the usual way.
```

Reply:
Balls. Of course, the preceding bullet doesn't work for Infantry that have Jump Packs. I'll add a new suggestion which covers all the lines together.

Reply:
I'll resolve this conversation, as we have a new conversation on all three bullet rather than just the latter two.